src/module-elasticsuite-catalog-optimizer/view/adminhtml/layout/smile_elasticsuite_catalog_optimizer_optimizer_duplicate.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```

Comment:
There should not be any new page layout for duplicate action, it's the same as the edit page.